generators/Generators.fs

```diff
@@ -621,6 +621,48 @@ type GoCounting() =
                     None            
         | _ -> None
 
+type GradeSchool() =
+    inherit GeneratorExercise()
+
+    member private this.RenderPropertyValue canonicalDataCase propertyName =
+        this.RenderInput(canonicalDataCase, propertyName, Map.find propertyName canonicalDataCase.Input)
+
+    member private __.RenderStudent (student: JToken) =
+        let values = student.Values<JToken>()
+        sprintf "(\"%s\",%s)" (student.First|>string) (student.Last|>string)
+
+    member private this.RenderArrangeList canonicalDataCase = 
```

Comment:
You should be able to simplify this code a bit. For inspiration, please check the existing [dominoes generator](https://github.com/exercism/fsharp/blob/master/generators/Generators.fs#L493).